components/shared/modals/ModalBody.vue

```diff
@@ -105,11 +114,21 @@ const skeletonTitle = computed(
 
 const onClose = () => emits('close')
 
+watch(
+  [() => props.loading, () => props.estimatedTime],
+  ([loading, estimatedTime]) => {
+    if (loading && Boolean(estimatedTime)) {
+      startEstimatedTime(Number(estimatedTime))
```

Comment:
why do you need to use `Number()`?

Reply:
![CleanShot 2024-04-15 at 12 30 59@2x](https://github.com/kodadot/nft-gallery/assets/44554284/985416f1-e9a9-4299-bf21-e28d12cd45ce)



Number(undefined) = false
Number(0) = false

shouldn't it be only number at this point ?

Reply:
ic. ts stuff